src/sac_stac/service_layer/operations.py

```diff
@@ -0,0 +1,15 @@
+ISO = {
+    'fiji' : 'FJI',
+    'solomon Islands' : 'SLB',
+    'vanuatu' : 'VUT',
+}
+
+def get_iso(country: str) -> str:
+    """
+    Return ISO code based on country
+
+    :param country: Country name (str)
+
+    :return: ISO Code (str)
+    """
+    return ISO.get(country.strip().lower())
```

Comment:
doing lower here but solomon islands has an upper case 'i' in the dictionary definition above